css/index.css

```diff
@@ -111,6 +130,36 @@ p {
     align-items: center;
     justify-content: center;
 }
+.middle-content .boxes .box1{
```

Comment:
This could have been done as nth child as well. If I were to make specific classes for each box I'd want the classes to be reusable elsewhere.

Also your selector doesn't need to be that specific unless you are using the "box1" class elsewhere. You can just use the box1 class selector.